torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -767,7 +767,7 @@ StmtPtr TensorExprKernel::transformLoops(BackendType backendType, StmtPtr st) {
   if (pre_alloc_) {
     auto interm_bufs = l.getIntermediateBufs();
     preAllocIntermediateBufs(interm_bufs);
-    l.prepareForCodegen(interm_bufs);
+    l.prepareForCodegen();
```

Comment:
Please move `l.prepareForCodegen()` out from the if and remove the `else` branch.